ntoskrnl/mm/freelist.c

```diff
@@ -318,7 +318,7 @@ MiAllocatePagesForMdl(IN PHYSICAL_ADDRESS LowAddress,
                 // Get the PFN entry for this page
                 //
                 Pfn1 = MiGetPfnEntry(Page);
```

Comment:
There's a MiGetPfnEntry call above too (line 282), maybe that one could benefit from the assert.

Reply:
No, because it is dereferenced right after that. If it's NULL, it'll crash.

Reply:
yeah but then one could ponder the same about that other one below:
https://github.com/reactos/reactos/pull/7737/files#diff-549b72fe8cc6cc8d709b3599cf35a72bbba96dc39c35bb127feac71fd8ef7526R410

Reply:
Feel free to ponder, if you like to. But I will neither add pointless asserts nor remove them in places that have nothing to do with this PR.